api/apps/api/src/modules/analysis/providers/planning-units/update-planning-units-events.port.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * API side cannot put other states
```

Comment:
very minor, but while leaving useful notes, I'd also add a tiny bit of rationale (because I myself won't remember even just next week, let alone a new dev coming across this at some point 😅)

```suggestion
 * API side cannot set other states: `Finished` will only be set
    by the job processor handling this task.
```